Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -927,6 +930,18 @@ function vtkRenderWindowInteractor(publicAPI, model) {
       }
     }
   };
+
+  // Stop animating if the renderWindowInteractor is deleted.
+  const superDelete = publicAPI.delete;
+  publicAPI.delete = () => {
+    if (animationRequesters) {
+      animationRequesters.forEach((animationRequester) =>
+        publicAPI.cancelAnimation(animationRequester)
+      );
+      model.animationRequest = null;
```

Comment:
I do not think you need to reset `animationRequest` to null, it will be done automatically when cancelAnimation is called for the last requester.